docs/Api/HoldingsApi.md

```diff
@@ -0,0 +1,180 @@
+# atrium\HoldingsApi
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**listHoldings**](HoldingsApi.md#listHoldings) | **GET** /users/{user_guid}/holdings | List holdings
+[**listHoldingsByAccount**](HoldingsApi.md#listHoldingsByAccount) | **GET** /users/{user_guid}/accounts/{account_guid}/holdings | List holdings by account
+[**listHoldingsByMember**](HoldingsApi.md#listHoldingsByMember) | **GET** /users/{user_guid}/members/{member_guid}/holdings | List holdings by member
+[**readHolding**](HoldingsApi.md#readHolding) | **GET** /users/{user_guid}/holdings/{holding_guid} | Read holding
+
+
+# **listHoldings**
+> \atrium\model\HoldingsResponseBody listHoldings($user_guid)
+
+List holdings
```

Comment:
This is a little nitpicky, but are we sure we want to use the `list` naming scheme? Usually I would expect it to match the REST verb, which would make it `getHoldings`

Reply:
That's a good point. I think we will want to keep `list` at least in this MR to stay consistent with the other endpoints in the library that use `list`.

Besides using `list`, we have also been using `read` when getting singular resources. That also may be something we also look at in a future iteration.